### PR TITLE
Add CI support

### PR DESCRIPTION
**Pull request by RobiNino**
- [ ] All [tests](https://github.com/jfrog/jfrog-vscode-extension#building-and-testing-the-sources) passed. If this feature is not already covered by the tests, I added new tests.
- [x] I used `npm run format` for formatting the code before submitting the pull request.
-----

Depends on [jfrog/jfrog-client-js/pull/27](https://github.com/jfrog/jfrog-client-js/pull/27) and currently using the package that adds that locally.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     },
     "publisher": "JFrog",
     "engines": {
-        "vscode": "^1.37.0"
+        "vscode": "^1.56.2"
```
Comment (yahavi) [inline package.json:18]: What is the reason for updating the minimal version to 1.56.2?

File: package.json [hunk 2]
```diff
@@ -164,17 +206,7 @@
...
-        "jsonValidation": [
-            {
-                "fileMatch": [
-                    "**/filespecs/*.json",
-                    "*filespec*.json",
-                    "*.filespec"
-                ],
-                "url": "https://github.com/jfrog/jfrog-cli/raw/master/schema/filespec-schema.json"
-            }
-        ]
```
Comment (yahavi) [inline package.json:177]: Please put that back

File: README.md [hunk 3]
```diff
@@ -39,8 +39,6 @@ The JFrog VS Code Extension adds JFrog Xray scanning of project dependencies to
 It allows developers to view panels displaying vulnerability information about the components and their dependencies directly in their VS Code IDE.
 With this information, a developer can make an informed decision on whether to use a component or not before it gets entrenched into the organization’s product.
 
-The extension also applies [JFrog File Spec JSON schema](https://raw.githubusercontent.com/jfrog/jfrog-cli/master/schema/filespec-schema.json) on the following file patterns: `**/filespecs/*.json`, `*filespec*.json` and `*.filespec`. Read more about JFrog File specs [here](https://www.jfrog.com/confluence/display/JFROG/FileSpec).
-
```
Comment (yahavi) [inline README.md:43]: Please put that back

File: README.md [hunk 4]
```diff
@@ -212,14 +230,41 @@ Behind the scenes, the extension builds the Pypi dependencies tree by running `p
...
+- Security information about the build artifacts and dependencies
+
+### How Does It Work?
+The CI information displayed in VS Code is pulled by the JFrog Extension directly from JFrog Artifactory. This information is stored in Artifactory as part of the build-info, which is published to Artifactory by the CI server. 
+
+Read more about build-info in the [Build Integration](https://www.jfrog.com/confluence/display/JFROG/Build+Integration) documentation page. If the CI pipeline is also configured to scan the build-info by JFrog Xray, the JFrog VS Code Extension will pull the results of the scan from JFrog Xray and display them in the CI view as well.
+
+Read the [Setting up CI Integration](https://www.jfrog.com/confluence/display/JFROG/JFrog+IntelliJ+IDEA+Plugin#JFrogIntelliJIDEAPlugin-SettingupCIintegration) for more information.
+
+### Displaying Build Information
```
Comment (eyalbe4) [inline README.md:261]: Displaying Build Information --> Setting Up the CI View
(let's also update the table of content).

File: README.md [hunk 5]
```diff
@@ -212,14 +230,41 @@ Behind the scenes, the extension builds the Pypi dependencies tree by running `p
...
+- Git branch and latest commit message
+- Link to the CI run log
+- Security information about the build artifacts and dependencies
+
+### How Does It Work?
+The CI information displayed in VS Code is pulled by the JFrog Extension directly from JFrog Artifactory. This information is stored in Artifactory as part of the build-info, which is published to Artifactory by the CI server. 
+
+Read more about build-info in the [Build Integration](https://www.jfrog.com/confluence/display/JFROG/Build+Integration) documentation page. If the CI pipeline is also configured to scan the build-info by JFrog Xray, the JFrog VS Code Extension will pull the results of the scan from JFrog Xray and display them in the CI view as well.
+
+Read the [Setting up CI Integration](https://www.jfrog.com/confluence/display/JFROG/JFrog+IntelliJ+IDEA+Plugin#JFrogIntelliJIDEAPlugin-SettingupCIintegration) for more information.
```
Comment (eyalbe4) [inline README.md:259]: Let's replace this "read more" section with a section titled as "Setting Up your CI Pipeline".
Here's the content we can add for this section:

Before VS Code can display information from your CI in the CI View, your CI pipeline needs to be configured to expose this data. Read [this guide] which describes how to configure your CI pipeline.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -36,25 +42,11 @@ The earlier you remediate a vulnerability in the release cycle, the lower the co
 or even sooner, during the development.
 
 The JFrog VS Code Extension adds JFrog Xray scanning of project dependencies to your VS Code IDE.
-It allows developers to view panels displaying vulnerability information about the components and their dependencies directly in their VS Code IDE.
-With this information, a developer can make an informed decision on whether to use a component or not before it gets entrenched into the organization’s product.
+It allows developers to view panels displaying vulnerability information about the components and their dependencies directly in their VS Code IDE. The information can be both about the current project developed, and about builds on the CI system.
```
Comment (eyalbe4) [inline README.md:46]: Let's add the following paragraph here - 

The extension also allows developers to track the status of the code while it is being built, tested and scanned on the CI server.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -212,14 +231,43 @@ Behind the scenes, the extension builds the Pypi dependencies tree by running `p
...
+- Git branch and latest commit message
+- Link to the CI run log
+- Security information about the build artifacts and dependencies
+
+### How Does It Work?
+The CI information displayed in VS Code is pulled by the JFrog Extension directly from JFrog Artifactory. This information is stored in Artifactory as part of the build-info, which is published to Artifactory by the CI server. 
+
+Read more about build-info in the [Build Integration](https://www.jfrog.com/confluence/display/JFROG/Build+Integration) documentation page. If the CI pipeline is also configured to scan the build-info by JFrog Xray, the JFrog VS Code Extension will pull the results of the scan from JFrog Xray and display them in the CI view as well.
+
+#### Setting Up Your CI Pipeline
```
Comment (eyalbe4) [inline README.md:260]: #### --> ###
(Let's also update this section's table of content level accordingly).

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

**Pull request merged**